Content/IsBlockCapture.md

```diff
@@ -0,0 +1,13 @@
+# Is Block Capture
+
+_IsBlockCapture_ is a sandbox setting in [Sandboxie Ini](SandboxieIni.md). If enabled it will prevent sandboxed processes from accessing the images of the window outside the sandbox. For example:
```

Comment:
```suggestion
_IsBlockCapture_ is a sandbox setting in [Sandboxie Ini](SandboxieIni.md). If enabled, it will prevent sandboxed processes from accessing the images of the window outside the sandbox. For example:
```